src/Http/Controllers/StaticController.php

```diff
@@ -22,9 +23,8 @@ public function index(Request $request): BinaryFileResponse
         if (!File::exists($path) || $this->filesFilter(File::basename($path))) {
             abort(Response::HTTP_NOT_FOUND);
         }
-
         $headers = [
-            "Content-Type" => File::type($path),
+            "Content-Type" => File::extension($path) === self::FILE_CSS_EXTENSION ? null : File::type($path),
```

Comment:
```suggestion
            "Content-Type" => File::extension($path) === static::FILE_CSS_EXTENSION ? null : File::type($path),
```

Reply:
Also: I don't like narrowing to for CSS files only. Maybe renamed const should be an array of strings/types?